packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -92,7 +92,6 @@ export class HostedPluginReader implements BackendApplicationContribution {
         if (!pluginPath) {
             return undefined;
         }
-        pluginPath = path.normalize(pluginPath + '/');
```

Comment:
not sure about removing `normalize` that's to get rid of `..` and `.` in the path, don't think it is related to trailing slashes

Reply:
Is it better to normalize the path, but without trailing slash?
just
```
pluginPath = path.normalize(pluginPath);
```

Reply:
Checked the code again. I think the caller of `readPackage` must care about the correct path.

Reply:
@amiramw Could you check please that it does not introduce regressions for https://github.com/eclipse-theia/theia/pull/5274? Generally we should use portable URI here to abstract away from OS quirks, but if it does not break i'm fine to merge like that.